README.md

```diff
@@ -1 +1,111 @@
-# nested-hashes-intro
+# Introduction to Nested Hashes
+
+1. Understand what a nested hash is and why we use them. 
+2. Become familiar with a few situations in which you will encounter nested hashes as a developer. 
+3. Become familiar with the structure of a nested hash. 
+
+## Introduction: Why Use Nested Hashes
+
+Nested hashes are used to store complex collections of data.
+
+So far, we've seen hashes that store values in associated keys. In the hashes we've built up until now, each key points to a single value. Hashes are so useful, however, because they can be multi-dimensional, or nested. A key in a hash can point to a value that is a *collection of objects*, i.e. an array or even another hash. 
+
+As programmers, we strive to write code that models the real-world. The programs we write serve a purpose––whether you're creating a simple command line game or an app to help hospitals manage patient data, our code is designed to do a real job, like run a game or communicate critical information. 
+
+### Where Will You Find Nested Hashes?
+
+Nested hashes are a very common way to store and operate on complex associated data in a program. You are likely to encounter them any time you find yourself working with a large collection of information. In particular, you will encounter these data structures when working with data you will pull from APIs. 
+
+API stands for "Application Programming Interface" and here refers to the way in which organizations, companies and governments will expose their data to the public for use. 
+
+New York City, for example, has a robust API called NYC Open Data. Developers can connect to this API to find information about city programs, public housing, parks, schools, construction, you name it. 
+
+When you send a request for data to such an API, the data you get back from them will be in the form of a nested hash that can contain information about thousands of records. 
+
+We'll learn a lot more about APIs later on in this course. For now, just understand that nested hashes are a very common occurrence in programming, they are used to store complex collections of data and you will encounter them in particular when working with APIs, among other places.
+
+### Nested Hashes Model Real-World Data
+
+We can imagine so many real-world situations and environments in which we are dealing with complicated collections of data. Take the example of a simple address book. 
```

Comment:
Tie the example into some sort of NYC OPen Data. Maybe like NYC Open Data has a directory of doctors or w/e and link to it. Trying to really tie it to the real wordl